StartEycaActivation/handler.ts

```diff
@@ -196,7 +196,7 @@ export function StartEycaActivationHandler(
                 )
         )
       ),
-      TE.chain(() =>
+      TE.chainW(() =>
```

Comment:
is this necessary?

Reply:
Yep, after the migration we get a compile error here